cmake/Version.cmake

```diff
@@ -2,6 +2,11 @@ set(CMAKE_VERSION_MAJOR 22)
 set(CMAKE_VERSION_MINOR 2)
 set(CMAKE_VERSION_PATCH 0)
 
+set(CMAKE_PROJECTED_RELEASE_DATE "30 September 2022")
+
+set(PREV_RELEASE_MAJOR 22)
+set(PREV_RELEASE_MINOR 1)
+set(PREV_RELEASE_PATCH 0)
```

Comment:
Added a few more variables right here in a very convenient place.  When we update versions we are already in here specifying the new version numbers, so this is the best spot.